src/client/assets/line/lineSvgs.js

```diff
@@ -0,0 +1,440 @@
+        '<path fill="none" stroke="#000000" d="M5,10.5L45,10.5" stroke-width="1" stroke-dasharray="0" ' +
+        'style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"></path>' +
+        '<text x="25" y="9" text-anchor="middle" font="10px &quot;Arial&quot;" stroke="none" fill="#000000" ' +
+        'font-size="8px" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0); text-anchor: middle; ' +
+        'font-style: normal; font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 8px; ' +
+        'line-height: normal; font-family: Arial;"><tspan dy="9" ' +
+        'style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">$</tspan></text></svg>';
+    line_svg_directory[REGISTRY_KEYS.LINE_LABEL_PLACEMENT][CONSTANTS.LINE_STYLE.LABEL_PLACEMENTS.SRC] =
+        '<svg height="20" version="1.1" width="50" xmlns="http://www.w3.org/2000/svg" ' +
+        'style="overflow: hidden; position: relative;">' +
+        '<desc style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">Created with Raphaël 2.1.2</desc>' +
+        '<defs style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"></defs>' +
+        '<path fill="none" stroke="#000000" d="M5,10.5L45,10.5" stroke-width="1" stroke-dasharray="0" ' +
+        'style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"></path><text x="7" y="9" text-anchor="middle" ' +
+        'font="10px &quot;Arial&quot;" stroke="none" fill="#000000" font-size="8px" ' +
+        'style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0); text-anchor: middle; font-style: normal; ' +
+        'font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 8px; line-height: normal; ' +
+        'font-family: Arial;"><tspan dy="9" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">$</tspan></text><' +
+        '/svg>';
+    line_svg_directory[REGISTRY_KEYS.LINE_LABEL_PLACEMENT][CONSTANTS.LINE_STYLE.LABEL_PLACEMENTS.DST] =
+        '<svg height="20" version="1.1" width="50" xmlns="http://www.w3.org/2000/svg" ' +
+        'style="overflow: hidden; position: relative;">' +
+        '<desc style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">Created with Raphaël 2.1.2</desc>' +
+        '<defs style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"></defs><path fill="none" ' +
+        'stroke="#000000" d="M5,10.5L45,10.5" stroke-width="1" stroke-dasharray="0" ' +
+        'style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);"></path>' +
+        '<text x="43" y="9" text-anchor="middle" font="10px &quot;Arial&quot;" stroke="none" fill="#000000" ' +
+        'font-size="8px" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0); text-anchor: middle; ' +
+        'font-style: normal; font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 8px; ' +
+        'line-height: normal; font-family: Arial;"><tspan dy="9" ' +
+        'style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">$</tspan></text></svg>';
+
+    return line_svg_directory;
+});
+
+/*
```

Comment:
have not added the bezier options yet so I figured that instead of debugging again we can keep it for a while, but I will attach a TODO if that is fine